### PR TITLE
Add Job lables getter

### PR DESCRIPTION
**Pull request by orlandohohmeier**
/cc @jfurrow

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @kennyt

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

File: src/js/structs/Job.js [hunk 1]
```diff
@@ -18,6 +18,10 @@ module.exports = class Job extends Item {
     return this.get('id');
   }
 
+  getLabels() {
+    return this.get('labels');
```
Comment (aldipower) [inline src/js/structs/Job.js]: Just wondering, no default object needed here?
```
return this.get('labels') || {};
```
?

**Comment by kennyt** [comment]
conflicts

**Comment by rcorral** [comment]
Merge after rebase :)

File: src/js/structs/Job.js (hunk 1 above)
Comment (orlandohohmeier) [inline src/js/structs/Job.js]: Good catch! I'll add a default.

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Comment by orlandohohmeier** [comment]
**mesosphere-ci** retest this please

**Pull request merged**